lib/fog/share/openstack.rb

```diff
@@ -0,0 +1,239 @@
+module Fog
+  module Share
```

Comment:
To match a full generic name I think the module name `Share` would be something like SharedFileSystem. What do you think?